integration/e2emimir/services.go

```diff
@@ -178,14 +178,17 @@ func getBinaryNameForBackwardsCompatibility() string {
 	return "mimir"
 }
 
-func NewQueryFrontend(name string, flags map[string]string, options ...Option) *MimirService {
+func NewQueryFrontend(name string, consulAddress string, flags map[string]string, options ...Option) *MimirService {
```

Comment:
maybe mention that you only need a valid consul address if you're running ingest storage

Reply:
Sure, done in https://github.com/grafana/mimir/pull/8988/commits/5e3dbc02394f3d6288762faac3bf1da4fd839e9d